cmd/lassie/fetch.go

```diff
@@ -47,13 +48,19 @@ var fetchCmd = &cli.Command{
 			Usage:   "print progress output",
 		},
 		&cli.StringFlag{
-			Name:        "provider",
-			DefaultText: "The provider will be discovered automatically",
-			Usage:       "The provider addr including its peer ID. Example: /ip4/1.2.3.4/tcp/1234/p2p/12D3KooWBSTEYMLSu5FnQjshEVah9LFGEZoQt26eacCEVYfedWA4",
+			Name:        "providers",
```

Comment:
```suggestion
			Name:        "providers",
			Aliases:     []string{"provider"},
```

.. maybe? to be gentle, and don't we have docs for this somewhere already?